content/rc/subscriptions/upgrade-fixed-flexible.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: Upgrade subscription plan from Fixed to Flexible
+LinkTitle: Upgrade from Fixed to Flexible
+description: Upgrade your fixed subscription to a flexible subscription.
+weight: 45
+alwaysopen: false
+categories: ["RC"]
+aliases: 
+---
+
+Fixed subscription plans support low throughput workflows. They support a range of availability, persistence, and backup options, and can be great for testing and prototyping. However, if you notice that your databases need higher throughput, or you're missing features that are not available with fixed plans, you may want to upgrade your fixed plan to a flexible plan.
+
+For more information about the different subscription plans, see [Subscription plans]({{<relref "/rc/subscriptions#subscription-plans">}}).
+
+To upgrade your fixed plan to a higher subscription tier, see [Upgrade subscription plan]({{<relref "/rc/subscriptions/view-fixed-subscription#upgrade-subscription-plan">}}).
+
+## Upgrade fixed subscription to flexible
+
+To follow the steps in this guide, you must have a [fixed subscription]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) that you want to upgrade to a flexible subscription.
+
+To upgrade your fixed subscription to a flexible subscription:
+
+1. [Create a new flexible subscription](#create-flexible-subscription) with the right specifications to be able to migrate your databases.
+
+1. [Migrate your databases](#migrate-databases) from your fixed subscription to your new flexible subscription.
+
+1. [Delete your fixed subscription](#delete-fixed-subscription).
+
+### Create flexible subscription
+
+[Create a new flexible subscription]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) with the following specifications:
+
+- Select the **Version** that matches the Redis version your fixed subscriptions use.
+- In the [**Sizing tab**]({{<relref "/rc/subscriptions/create-fixed-subscription#sizing-tab">}}), create your databases with the following specifications:
+    - Set the memory limit to comply with [Active-Passive memory requirements]({{<relref "/rc/databases/migrate-databases#active-passive-memory-requirements">}}) if you want to migrate your database using [Active-Passive]({{<relref "/rc/databases/migrate-databases#sync-using-active-passive">}}).
```

Comment:
Aren't we supposed to use "Replica Of" instead of "Active-Passive"?

Reply:
Not on Cloud - it's still called Active-Passive in the UI.